### DISCUSSION TITLE
no download video, error message:.-(

### DISCUSSION THREAD

**Discussion by jozorr** [aclap-dev/video-downloadhelper#707] [Help and Q&A]
### Your question

Show me detected videos but no anle to download any, after a minute ended with this error message

### Addon & CoApp version and details

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "undef",
  "error": {
    "details": "SideDownload error: : [tcp @ 000001afaaaab000] Connection to tcp://ttc-cdn3-pub.mtv.cdn.moderntv.eu:443 failed: Error number -138 occurred\r\n[in#0 @ 000001afac26b880] Error opening input: Error number -138 occurred\r\nError opening input file https://ttc-cdn3-pub.mtv.cdn.moderntv.eu/moderntv/stream/dvojka/40-hls/storage-media.m3u8?_cdn_attrs=account%3Dmoderntv%2Cresource%3Ddvojka_stream_5n&_cdn_session=66354366161&_cdn_timestamp=1717232313&_cdn_token=fb9de5a742ce838bfd9ef43cf6a69531559c5c8d&duration=6300&start=1716756000.\r\nError opening input files: Error number -138 occurred\r\n",
    "downloadable_id": "downloadable_1708003126504369",
    "error": "coapp_failure",
    "id": "coapp_failure_033ba261-009d-46e0-a476-7bb7b7675f12"
  }
}

**Comment by GrampaWildWilly** [comment]
You'll need to post the URL of a web page hosting a video your are having a problem with.  Don't just say it's a web site.  Post the specific URL of a specific video.

**Reply by jozorr** [reply]
for example this page. https://sledovanietv.sk/home#record%3A596147672
but you need an accoint to run a video on it

**Reply by GrampaWildWilly** [reply]
In that case, I can't help you.

**Comment by Mattmjs** [comment]
I looked at this yesterday using the m3u8 link in a new tab and it appears to work for me. I got a few minutes before stopping it.
Doesn't look like it has expired yet, I tried in Firefox. It looks like a stage show and is 1:45:00 in duration.

**Comment by jozorr** [comment]
Hi,  I found out that is due to PROXY server. Have you any idea how can I bypass or add some exception?

Odoslané z aplikácie Pošta<https://go.microsoft.com/fwlink/?LinkId=550986> pre Windows

________________________________
Od: Mattmjs ***@***.***>
Odoslané: Tuesday, May 28, 2024 9:06:24 AM
Komu: aclap-dev/video-downloadhelper ***@***.***>
Kópia: jozorr ***@***.***>; Author ***@***.***>
Predmet: Re: [aclap-dev/video-downloadhelper] no download video, error message:.-( (Discussion #707)


I looked at this yesterday using the m3u8 link in a new tab and it appears to work for me. I got a few minutes before stopping it.
Doesn't looked like it has expired yet, I tried in Firefox. It looks like a stage show and is 1:45:00 in duration.

**Comment by Mattmjs** [comment]
No solution on that at the moment, use a vpn if possible.
